staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config_test.go

```diff
@@ -928,6 +1003,14 @@ func TestIsKMSv2ProviderHealthyError(t *testing.T) {
 				Version: "v2alpha1",
 			},
 		},
+		{
```

Comment:
This is pre-existing but can you fix this test to actually check what the error message is instead of just a `nil` check.